engines/static/app/controllers/static/contacts_controller.rb

```diff
@@ -0,0 +1,62 @@
+require_dependency "static/application_controller"
+
+module Static
+  class ContactsController < ApplicationController
```

Comment:
You added this contacts controller, I didn't understand:
You were supposed to add static pages controller for content management.

Reply:
Company website has a ContactUs Page so tried to create Content Management for ContactUs page. Its somewhat similar to helpdesk which you gave as an example.